spec/controllers/works_controller_spec.rb

```diff
@@ -1060,4 +1060,33 @@
       end
     end
   end
+
+  describe "#assign_curator" do
+    let(:errors) { double }
+    let(:current_work) { instance_double(Work) }
+    let(:params) do
+      {
+        id: current_work.id,
+        uid: "test_user"
+      }
+    end
+
+    context "when the request parameters are valid" do
```

Comment:
I think this should have been invalid?
```suggestion
    context "when the request parameters are invalid" do
```